This patch adds the text/plain MIME type as a supported MIME type for
the library to process files with.
PR: https://github.com/hughsie/libxmlb/pull/9

diff --git a/src/xb-builder-source.c b/src/xb-builder-source.c
index a2f7f42..38c7025 100644
--- a/src/xb-builder-source.c
+++ b/src/xb-builder-source.c
@@ -509,7 +509,7 @@ xb_builder_source_init (XbBuilderSource *self)

 	/* built-in types */
 	xb_builder_source_add_converter (self,
-					 "application/xml",
+					 "application/xml,text/plain",
 					 xb_builder_source_load_plain_cb,
 					 NULL, NULL);
 	xb_builder_source_add_converter (self,
